api/src/identity-access-management/infrastructure/repositories/anonymous-user-token.repository.js

```diff
@@ -0,0 +1,29 @@
+import crypto from 'node:crypto';
+
+import { config } from '../../../shared/config.js';
+import { temporaryStorage } from '../../../shared/infrastructure/key-value-storages/index.js';
+
+const anonymousUserTokenTemporaryStorage = temporaryStorage.withPrefix('anonymous-user-token:');
+
+/**
+ * @param {*} userId User ID of the anonymous user
+ * @returns {Promise<string|null>} Token as UUID or null if not found
+ */
+const get = (userId) => {
+  return anonymousUserTokenTemporaryStorage.get(userId);
+};
```

Comment:
D’après les standards de l’équipe, cette fonction qui renvoie `null` lorsque l’utilisateur n’est pas trouvé, cette fonction ne devrait-elle pas être un `find` plutôt qu’un `get` ?
```suggestion
const find = (userId) => {
  return anonymousUserTokenTemporaryStorage.get(userId);
};
```